cmd/dexctl/command_config.go

```diff
@@ -81,10 +82,36 @@ func runGetConnectorConfigs(cmd *cobra.Command, args []string) int {
 	fmt.Printf("Found %d connector config(s)\n", len(cfgs))
 
 	for _, cfg := range cfgs {
-		fmt.Println()
-		fmt.Printf("ID:   %v\n", cfg.ConnectorID())
-		fmt.Printf("Type: %v\n", cfg.ConnectorType())
+		switch typedCFG := cfg.(type) {
```

Comment:
I'm a little confused by what this is doing and why it's necessary. Can you expand a little bit?